experiments/ClimaEarth/setup_run.jl

```diff
@@ -681,29 +668,25 @@ function step!(cs::CoupledSimulation)
     !isnothing(cs.conservation_checks) && ConservationChecker.check_conservation!(cs)
     ClimaComms.barrier(comms_ctx)
 
-    ## update water albedo from wind at dt_water_albedo
-    ## (this will be extended to a radiation callback from the coupler)
-    TimeManager.maybe_trigger_callback(cs.callbacks.water_albedo, cs)
-
-    ## update the surface fractions for surface models,
-    ## and update all component model simulations with the current fluxes stored in the coupler
-    FieldExchanger.update_surface_fractions!(cs)
-    FieldExchanger.update_model_sims!(cs.model_sims, cs.fields)
-
     ## step component model simulations sequentially for one coupling timestep (Δt_cpl)
     FieldExchanger.step_model_sims!(cs.model_sims, cs.t[])
 
-    ## update the coupler with the new surface properties and calculate the turbulent fluxes
-    FieldExchanger.import_combined_surface_fields!(cs.fields, cs.model_sims) # i.e. T_sfc, surface_albedo, z0, beta
-    ## calculate turbulent fluxes in surfaces and save the weighted average in coupler fields
-    FluxCalculator.turbulent_fluxes!(cs.model_sims, cs.fields, cs.boundary_space, cs.thermo_params)
-
-    Interfacer.update_field!(atmos_sim, Val(:surface_temperature), cs.fields)
-
+    ## update the surface fractions for surface models
+    FieldExchanger.update_surface_fractions!(cs)
 
+    ## update the coupler with the new surface properties
+    FieldExchanger.import_combined_surface_fields!(cs.fields, cs.model_sims)
     ## update the coupler with the new atmospheric properties
-    FieldExchanger.import_atmos_fields!(cs.fields, cs.model_sims) # radiative and/or turbulent
+    FieldExchanger.import_atmos_fields!(cs.fields, cs.model_sims)
+    ## update the model simulations with the new coupler fields
+    FieldExchanger.update_model_sims!(cs.model_sims, cs.fields)
+
+    ## calculate turbulent fluxes in the coupler and update the model simulations with them
+    FluxCalculator.turbulent_fluxes!(cs.model_sims, cs.fields, cs.boundary_space, cs.thermo_params)
 
+    ## update water albedo from wind at dt_water_albedo
+    ## (this will be extended to a radiation callback from the coupler)
+    TimeManager.maybe_trigger_callback(cs.callbacks.water_albedo, cs)
```

Comment:
Why is this needed now?

Reply:
Before, it was called at the beginning of each step. I moved it here so it's called alongside the checkpoint callback. I don't think it matters when during a step we update the albedo since we update it so often anyway

Reply:
ok!